inc/class_core.php

```diff
@@ -519,6 +521,8 @@ public function get_asset_url($path = '', $use_cdn = true)
 			$url = $path;
 		}
 
+		$plugins->run_hooks('get_asset_url_end', $url);
```

Comment:
> And how would you suggest implementing this then?

I wouldn't, TBH.

That is why I'm seeking for more feedback, it is not up to us only really..